src/applications/simple-forms/21-10210/utils.js

```diff
@@ -31,11 +33,11 @@ export const witnessHasOtherRelationship = formData => {
 };
 
 export const getFocusSelectorFromPath = pathname => {
-  let focusSelector = '#nav-form-header';
+  let focusSelector = defaultFocusSelector;
```

Comment:
I think there is specifically a unit test that is looking for this that will need to be changed.

Reply:
Unit-test refactored.